### DISCUSSION TITLE
occupancy counting

### DISCUSSION THREAD

**Discussion by dansrogers** [mKeRix/room-assistant#883] [Ideas]
It would be great if room-assistant is able to count occupants of a room with just a single sensor per door.  If you count entry/exit to a room then you can have a highly count of the total people in the room.  This can reduce the number and accuracy of thermal sensors needed per room.

**Comment by mKeRix** [comment]
Was originally requested in #159 - would be happy to see it added to room-assistant & can help with any PRs for this.
